src/vscode-dts/vscode.d.ts

```diff
@@ -15665,6 +15680,13 @@ declare module 'vscode' {
 		 */
 		forEach(callback: (item: TestItem, collection: TestItemCollection) => unknown, thisArg?: any): void;
 
+		/**
+		 * Get a new iterator with `[test item, test item]` pairs from the collection.
+		 *
+		 * Node that the two elements are the same here to be compatible with `new Map().entries()`
+		 */
+		entries(): IterableIterator<[TestItem, TestItem]>;
```

Comment:
This matches how `new Set().entries()` works. We can decide if this makes sense if we should just do the more natural seeming `IterableIterator<TestItem>`

Reply:
This also acts as and is backed by a map (see `.get()` below), so returning `[string /* itemId */, TestItem]` would make sense.

Reply:
@connor4312 Thanks. Can you check out the latest changes too. They switch to `[Symbol.iterator]` instead and I want to make sure iterating through key-value pairs still makes sense (instead of just iterating through the test items)